src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1104,23 +1103,11 @@ public ImmutableArray<SymbolDisplayPart> ToDisplayParts(SymbolDisplayFormat form
             CSharpSyntaxNode block,
             CSharpSyntaxNode expression,
             CSharpSyntaxNode syntax,
-            DiagnosticBag diagnostics)
+            DiagnosticBag diagnostics,
+            ErrorCode code)
```

Comment:
I considered that.  However, i don't like defaults because it's far too easy for someone to forget to pass it along (i've encountered more bugs than i can count because of this sort of approach).  The current approach forces callers to think about what the right code is, and ensures that the right behavior will be there.